x-pack/plugins/index_management/public/application/components/mappings_editor/__jest__/client_integration/datatypes/scaled_float_datatype.test.tsx

```diff
@@ -90,15 +89,12 @@ describe.skip('Mappings editor: scaled float datatype', () => {
     await act(async () => {
       form.setInputValue('scalingFactor.input', '123');
     });
+
+    component.update();
+
     await updateFieldAndCloseFlyout();
-    expect(exists('mappingsEditorFieldEdit')).toBe(false);
 
-    if (kibanaVersion.major < 7) {
```

Comment:
This PR will only be merged into `main` so we no longer need to test for the `boostParameter` toggle.